services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py

```diff
@@ -0,0 +1,118 @@
+import random
+# from src import db
+import pandas as pd 
+import numpy as np
+from collections import defaultdict
+# from sqlalchemy.sql import text
+import os
+import pickle
+import sys
+import subprocess
+from .AbstractModel import AbstractModel
+
+
+class AlphaModel(AbstractModel):
+    def predict_probabilities(self, content_ids, user_id, seed=None, **kwargs):
+        
+        try:
```

Comment:
nit: this try-except block is too large, which line do you think will have the exception, only put those lines in the try-catch or refactor into a function call